pkg/network/ebpf/c/cookie.h

```diff
@@ -1,12 +1,15 @@
 #ifndef __COOKIE_H__
 #define __COOKIE_H__
 
-#include <linux/types.h>
-#include <net/sock.h>
+#include "ktypes.h"
 
 static __always_inline u32 get_sk_cookie(struct sock *sk) {
+#ifdef COMPILE_CORE
+    return bpf_get_prandom_u32();
```

Comment:
Why is this only used in CO-RE mode?

Reply:
Switched to using the `bpf_helper_exists` function.

Reply:
This is available in 4.1. Is the concern for CentOS/RHEL?

Reply:
Hm... I didn't pay attention to how low the id for this function is. So it really should be available across the kernels we support. Took away all conditional compilation, so we're just calling `bpf_get_prandom_u32` across all variants.

Reply:
👍